src/applications/third-party-app-directory/components/SearchResult/index.jsx

```diff
@@ -49,7 +49,10 @@ export class SearchResult extends Component {
     };
 
     return (
-      <li className="third-party-app vads-u-display--flex vads-u-flex-direction--column vads-u-margin-bottom--2 vads-u-padding--3 vads-u-border-color--gray-lightest vads-u-border--2px">
+      <li
+        aria-live="polite"
```

Comment:
[This](https://dsva.slack.com/archives/CQH357ZTP/p1597243405017600?thread_ts=1597240608.005300&cid=CQH357ZTP) is a conversation we had about this attribute a while back: 
>Aria-live regions have to be in the DOM on first paint, or SRs don’t know to watch them.

Which issue is this change resolving?

Reply:
This change was to resolve [16464](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16464). Specifically, the issue with ie11/JAWS skipping the expanded additional info content.

In my research I didn't see anything regarding first paint, just that the attribute should exist prior to any change, but if first paint is correct that would imply that any `aria-live` fix wouldn't work in this widget.

I haven't tested this particular change due to not having a JAWS license.

Reply:
So I was able to test the issue using a demo version of JAWS and the screen reader skipping expanded content seems to be resolved in production. It looks like adding the `aria-expanded` attribute in my previous PR helped give context to the screen reader. Removed `aria-live` in my latest commit.